data/modules/TradeShips/Events.lua

```diff
@@ -58,7 +58,7 @@ local onEnterSystem = function (ship)
 	elseif Core.ships[ship] ~= nil then
 		local trader = Core.ships[ship]
 		Core.log:add(ship, 'Entered '..Game.system.name..' from '..trader.from_path:GetStarSystem().name)
-
+		
```

Comment:
Please take a closer look; in many places you have tabs on an empty line - it would be nice to not allow this.

Reply:
Ok, I will try to do this on clear head, to not miss anything.

Reply:
There should be no more such errors.
Thanks to @impaktor for suggesting to install plugin for `editorconfig`-file.
P.S.: I use `Notepad++` for `lua` code. This also has built-in feature to trim trailing spaces on save.